### PR TITLE
GH-1022 Fix layout issues Not Found page

### PR DESCRIPTION
**Pull request by evdhoudt**
Closes NCI-Agency/anet#1022

### CODE REVIEW

File: client/src/components/ResponsiveLayout.js [hunk 1]
```diff
@@ -118,7 +118,12 @@ class ResponsiveLayout extends Component {
 						}}
 					/>
 					{(pageProps.useNavigation === true || floatingMenu === true) &&
-						<div style={sidebarContainer} className={ floatingMenu === false ? "hidden-xs" : "nav-overlay"}>
+						<div
+							style={sidebarContainer}
+							className={floatingMenu === false ? "hidden-xs" : "nav-overlay"}
+							name="mainSidebar"
```
Comment (gjvoosten) [inline client/src/components/ResponsiveLayout.js]: `name` attribute is not needed.

File: client/src/components/ResponsiveLayout.js [hunk 2]
```diff
@@ -118,7 +118,12 @@ class ResponsiveLayout extends Component {
...
+							id="main-sidebar"
```
Comment (gjvoosten) [inline client/src/components/ResponsiveLayout.js]: Using classes over id's is preferable.

File: client/src/components/NotFound.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default class NotFound extends Component {
 
 	render() {
 		return <div>
-            <h1 className="not-found-text">{this.props.text}</h1>
+            <h1 style={{ textAlign: 'center' }}>{this.props.text}</h1>
```
Comment (gjvoosten) [inline client/src/components/NotFound.js]: Why is this changed from a class to an in-line style?

File: client/src/components/NotFound.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import React, {Component} from 'react'
+import React, { PureComponent } from 'react'
 
-export default class NotFound extends Component {
+export default class NotFound extends PureComponent {
```
Comment (gjvoosten) [inline client/src/components/NotFound.js]: Why the change from Component to PureComponent?

File: client/src/index.css [hunk 5]
```diff
@@ -402,25 +402,10 @@ li#search-nav .nav > li a {
...
-	padding: 0;
-	margin: 0;
-}
-
-.not-found .not-found-text {
-	text-align: center;
-}
-
-.not-found .col-sm-3 {
+.not-found #main-sidebar {
```
Comment (gjvoosten) [inline client/src/index.css]: Use a class instead of an id?

File: client/src/components/ResponsiveLayout.js (hunk 1 above)
Comment (evdhoudt) [reply client/src/components/ResponsiveLayout.js]: Wanted to mirror mainViewport code for consistency. Should this be adjusted in both divs?

File: client/src/components/NotFound.js (hunk 3 above)
Comment (evdhoudt) [reply client/src/components/NotFound.js]: For styling it is preferred to use inline styles. This reduces dependencies to implicit style definitions and more..[https://github.com/necolas/react-native-web/blob/master/docs/guides/style.md](.https://github.com/necolas/react-native-web/blob/master/docs/guides/style.md)

File: client/src/components/ResponsiveLayout.js (hunk 1 above)
Comment (gjvoosten) [reply client/src/components/ResponsiveLayout.js]: There's no styling for `#main-viewport` so the id attribute could simply be removed (as could the name attribute).
[Strictly speaking, these changes do not pertain to the issue at hand, so at least make a separate commit with a clear message.]

File: client/src/components/NotFound.js (hunk 4 above)
Comment (evdhoudt) [reply client/src/components/NotFound.js]: Since it is a stateless component, this seemed more logical. Would prefer a functional component, but it messes with the global css state. (did not see a quick fix there)

Comment (gjvoosten) [reply client/src/components/NotFound.js]: Then I would prefer to see it in its own separate commit with a clear message, since it bears no direct relation to the referenced issue.

File: client/src/components/NotFound.js (hunk 3 above)
Comment (gjvoosten) [reply client/src/components/NotFound.js]: Then I would prefer to see it in its own separate commit with a clear message, since it bears no direct relation to the referenced issue.

File: client/src/components/ResponsiveLayout.js (hunk 1 above)
Comment (evdhoudt) [reply client/src/components/ResponsiveLayout.js]: #main-viewport is used in the scrollLinkNavItem, I will let it be for now.
I recall that the scroll-spy might also look at the name, I'm not quite sure, but there was something that might require the name property

Comment (gjvoosten) [reply client/src/components/ResponsiveLayout.js]: OK, but back to the original comment: _that_ div doesn't need a name.
